src/System.Private.CoreLib/shared/System/Text/UTF8Encoding.cs

```diff
@@ -264,24 +354,77 @@ public override unsafe int GetByteCount(ReadOnlySpan<char> chars)
         public override unsafe int GetBytes(char* chars, int charCount, byte* bytes, int byteCount)
         {
             // Validate Parameters
-            if (bytes == null || chars == null)
-                throw new ArgumentNullException(bytes == null ? nameof(bytes) : nameof(chars), SR.ArgumentNull_Array);
 
-            if (charCount < 0 || byteCount < 0)
-                throw new ArgumentOutOfRangeException((charCount < 0 ? nameof(charCount) : nameof(byteCount)), SR.ArgumentOutOfRange_NeedNonNegNum);
+            if (chars == null || bytes == null)
```

Comment:
`chars is null || bytes is null`?

Reply:
Same as https://github.com/dotnet/coreclr/pull/21948#discussion_r273688029. Doesn't compile.